src/collective/cover/layout.py

```diff
@@ -233,15 +233,12 @@ def update(self):
                 i += 1
 
 
-class Deco16Grid (grok.GlobalUtility):
-    grok.name('deco16_grid')
-    grok.implements(IGridSystem)
-
-    title = _(u'Deco (16 columns, default)')
-    ncolumns = 16
+class BaseGrid(object):
+    title = u''
+    ncolumns = 0
 
     row_class = 'row'
-    column_class = 'cell'
+    column_class = 'column'
```

Comment:
if grid is bootstrap, the cell class was making the grid to style wrong, and it was breaking many RF tests, so for bootstrap grid systems we use a new class for columns